mifosng-android/src/main/java/com/mifos/mifosxdroid/online/checkerinbox/CheckerInboxFragment.kt

```diff
@@ -398,7 +398,7 @@ class CheckerInboxFragment : MifosBaseFragment(), TextWatcher,
         val task = checkerTaskList.removeAt(position)
         fetchedCheckerTaskList.remove(task)
         checkerTaskListAdapter.submitList(checkerTaskList)
-        Toast.makeText(activity, getString(R.string.entry) + action, Toast.LENGTH_SHORT).show()
+        Toast.makeText(activity, getString(R.string.entry) + " " + action, Toast.LENGTH_SHORT).show()
```

Comment:
Use string formatter and declare a variable which contains the toastier message instead of passing it directly and using string concatenation.